FirebaseFirestoreSwift.podspec

```diff
@@ -36,5 +36,5 @@ Google Cloud Firestore is a NoSQL document database built for automatic scaling,
     'Firestore/third_party/FirestoreEncoder/*.swift',
   ]
 
-  s.dependency 'FirebaseFirestore', ">= 1.6.1"
+  s.dependency 'FirebaseFirestore', "~> 1.6.1"
```

Comment:
Should be `~> 1.6` to float minor version

Reply:
Done. My primary concern was to disallow interoperation with 1.6.0, which is untested.